io.kapsa.drive.metainfo.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.kapsa.drive</id>
+    <name>S3Drive</name>
+    <summary>Zero Knowledge E2E encrypted storage compatible with any S3 provider</summary>
+    <developer_name>S3Drive</developer_name>
+    <url type="homepage">https://s3drive.app</url>
+    <metadata_license>CC0</metadata_license>
+    <project_license>Proprietary</project_license>
+    <supports>
+        <control>pointing</control>
+        <control>keyboard</control>
+        <control>touch</control>
+    </supports>
+<!--  Search:   (.+)\n-->
+<!--  Replace:  <p>$1</p>\n-->
+    <description>
+        <p>S3Drive is an easy-to-use client that converts any S3 server into your personal encrypted file storage.</p>
+        <p>Your files are encrypted before they leave your device, so no one beside you can access them. Not even us.</p>
+
+        <p>Features:</p>
+        <p>Rclone compatible E2E encryption,</p>
+        <p>Filename/filepath encryption,</p>
+        <p>Photo &amp; video backup,</p>
+        <p>File versioning,</p>
+        <p>Secure sharing via encrypted link,</p>
+        <p>Desktop two-way sync,</p>
+        <p>Offline files,</p>
+        <p>ZIP download,</p>
+        <p>Multiple accounts support,</p>
+        <p>Search recursively,</p>
+        <p>Manage files &amp; directories (open, preview, download, copy, delete, rename, folder upload etc.),</p>
+        <p>Preview different file formats,</p>
+        <p>Edit text files,</p>
+        <p>Object Lock support,</p>
+        <p>Mount S3Drive as disk (desktop),</p>
+        <p>List and grid view,</p>
+        <p>Light and dark themes,</p>
+
+        <p>If you would like to participate in our shared journey, please visit Discord, Roadmap, or contact us directly.</p>
+
+        <p>Available on: Android, iOS, macOS, Windows, Linux, Web</p>
+        <p>Desktop clients: https://s3drive.app/desktop</p>
+        <p>Browser web client: https://web.s3drive.app</p>
+
+        <p>Support for AWS S3, Backblaze, Wasabi, Scaleway, Storj, MinIO and any other S3 compatible provider.</p>
+        <p>Sign-up for a free account, no credit card required.</p>
+
+        <p>Our roadmap: https://s3drive.canny.io/</p>
+        <p>Order by (name, size, modified),</p>
+        <p>Support streaming of encrypted videos,</p>
+        <p>Cross-client online sync,</p>
+        <p>Listing cache,</p>
+        <p>... and many many more, vote in our roadmap and stay tuned !</p>
+
+        <p>We would love to hear from you !</p>
+        <p>Missing some features? App not working as intended?</p>
+        <p>Please visit our Discord: https://discord.com/invite/YvxvypRqCu</p>
```

Comment:
don't put URL in descriptions. You can use the `<url>` tags for that.

Reply:
Moved out to: `<url type="contact">https://discord.com/invite/YvxvypRqCu</url>`